python/test/t_SimulatedAnnealingLHS_std.py

```diff
@@ -95,3 +96,14 @@
     print("  Intermediate design for restart iteration number ", i, design)
     print("  Final criteria: C2=%f, PhiP=%f, MinDist=%f" %
           (results.getC2(i), results.getPhiP(i), results.getMinDist(i)))
+
+# 5) Fix https://github.com/openturns/openturns/issues/1826
+optimalLHSAlgorithm = ot.SimulatedAnnealingLHS(
+    lhs, spaceFillingMinDist, geomProfile)
+print("lhs=", optimalLHSAlgorithm)
+design = optimalLHSAlgorithm.generate()
+print(
+    "Generating design using geometric temperature profile & MinDist criterion=", design)
+result = optimalLHSAlgorithm.getResult()
+print("Final criteria: C2=%f, PhiP=%f, MinDist=%f" %
+      (result.getC2(), result.getPhiP(), result.getMinDist()))
```

Comment:
I think `assert_almost_equal` can take Samples as arguments, so wouldn't it it be better than prints? Also, do we need a C++ equivalent for this test?

Anyway, thanks for solving issue #1826 so fast!

Reply:
By the way, you forgot to update the ChangeLog.

Reply:
Indeed for the ChangeLog, thanks
For the test I used the same structure as previous blocks. It is difficult to use `assert_almost_equal ` on `Sample` here (we do not know the exact optimal one). But we can possibly check final criteria (`C2`, `PhiP`, `MinDist`)

Reply:
Currently, you print the sample and check that the string of characters fits the .expout file. So you really are comparing the produced design with what OpenTURNS currently considers to be the optimal design, except in a non-numerical way. I do not see the difficulty in replacing that with an `assert_almost_equal`.

Reply:
If you are worried about float precision, you can adjust the tolerance, can't you?

Reply:
I don't want to store the output sample in the `py`. I will see
I found another issue(s) that I will first fix